### ISSUE TITLE
Add option to select from a list of names.

### ISSUE THREAD

**Issue by lordspacehog** [kitschysynq/befulnes#2]
Add cli option to select from a list of names.

**Comment by kitschysynq** [comment]
How would you want this to look? Would you have it just print a list of several names? Or would you want it to also prompt for a selection?

**Comment by lordspacehog** [comment]
I was thinking on it's own make it print a list and exit. Otherwise prompt
for a selection.

**Comment by kitschysynq** [comment]
So maybe a `-n NUM` parameter for the length of the list, and a `--interactive` flag to allow the choice from a list, leaving the remainder in the pool for later invocations?

**Comment by lordspacehog** [comment]
sounds decent enough, It'd be cool if the other flags still worked as well.

On Mon, Jun 15, 2015 at 1:17 PM, Sean Swehla notifications@github.com
wrote:

> So maybe a -n NUM parameter for the length of the list, and a
> --interactive flag to allow the choice from a list, leaving the remainder
> in the pool for later invocations?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/kitschysynq/befulnes/issues/2#issuecomment-112195698.

## 

MmMmMmMmM Pistol whip...

**Comment by kitschysynq** [comment]
Yeah, of course. So `-n NUM --project` would then initialize multiple repos, or something.
